src/components/Dialog-Window/Notes/DialogNote.module.scss

```
@@ -1,33 +1,48 @@
+form {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
 .textArea {
   resize: none;
-  position: absolute;
   width: 100%;
-  height: 100%;
+  height: 90%;
   font-size: inherit;
   font-family: inherit;
   padding-bottom: 2.8rem;
+  border: none;
+  outline: none;
 }
 
 .dynamicSavingText,
 .wordCounter {
-  bottom: -1.5rem;
   color: var(--theme-color-2);
   position: absolute;
 }
 
+.textAreaWrapper {
+  border: 1px solid black;
+  overflow: auto;
+  height: 100%;
+  position: relative;
+  top: 0.5rem;
```

Because we've removed the outline from `textarea`, we should add a way of showing that the element is focused. Adding it to the list

```suggestion
  top: 0.5rem;
  
  &::focus-within {
    // Add focus styling
  }
```